pkg/multiplexer/multiplexer.go

```diff
@@ -168,6 +170,7 @@ func (mux *Multiplexer) createTargetConn() net.Conn {
 			continue
 		}
 
+		time.Sleep(mux.delay)
```

Comment:
nit: run `time.Sleep` only if `mux.delay is > 0`, move it after the "new target connection" log and print a log line which indicates that the sleeping is happening.

Reply:
Done.